### PR TITLE
cloud: added integrations

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added integrations in TOC

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: TOC-tidb-cloud.md [hunk 1]
```diff
@@ -422,6 +412,17 @@
...
   - [Troubleshoot Inconsistency Between Data and Indexes](/troubleshoot-data-inconsistency-errors.md)
+- Integrations
+  - Terraform
+    - [TiDB Cloud Terraform Provider Overview](/tidb-cloud/terraform-tidbcloud-provider-overview.md)
+    - [Get TiDB Cloud Terraform Provider](/tidb-cloud/terraform-get-tidbcloud-provider.md)
+    - [Use Cluster Resource](/tidb-cloud/terraform-use-cluster-resource.md)
+    - [Use Backup Resource](/tidb-cloud/terraform-use-backup-resource.md)
+    - [Use Restore Resource](/tidb-cloud/terraform-use-restore-resource.md)
+  - [Vercel](/tidb-cloud/integrate-tidbcloud-with-vercel.md)
+  - Third-Party Monitoring Integrations
```
Comment (zhangyangyu) [inline TOC-tidb-cloud.md:423]: I think only `Monitoring` is enough, which means a category. `Third-Party` and `Integrations` are redundant here.

Comment (hfxsd) [inline TOC-tidb-cloud.md:423]:
```suggestion
  - Monitoring
```

**Review by zhangyangyu — APPROVED** [review]
How about put it before `Reference`? Others LGTM.

File: TOC-tidb-cloud.md [hunk 2]
```diff
@@ -422,6 +412,17 @@
   - [Dumpling](/dumpling-overview.md)
   - [Table Filter](/table-filter.md)
   - [Troubleshoot Inconsistency Between Data and Indexes](/troubleshoot-data-inconsistency-errors.md)
+- Integrations
```
[the remark this answers is not in the document]
Comment (hfxsd) [reply TOC-tidb-cloud.md:415]: Remove the Monitoring heading? What do you think @zhangyangyu

[the remark this answers is not in the document]
Comment (zhangyangyu) [reply TOC-tidb-cloud.md:415]: totally agree

**Review by zhangyangyu — APPROVED** [review]

File: tidb-cloud/third-party-integrations.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: Third-party Integrations
```
Comment (qiancai) [inline tidb-cloud/third-party-integrations.md:2]:
```suggestion
title: Third-party Monitoring Integrations
```

File: tidb-cloud/third-party-integrations.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+summary: Learn how to use third-party integrations.
```
Comment (qiancai) [inline tidb-cloud/third-party-integrations.md:3]:
```suggestion
summary: Learn how to use third-party monitoring integrations.
```

File: tidb-cloud/third-party-integrations.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+# Third-Party Integrations
```
Comment (qiancai) [inline tidb-cloud/third-party-integrations.md:6]:
```suggestion
# Third-Party Monitoring Integrations
```

File: tidb-cloud/third-party-integrations.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
```
Comment (qiancai) [inline tidb-cloud/third-party-integrations.md:7]:
```suggestion

You can integrate TiDB Cloud with third-party monitoring services to receive TiDB Cloud alerts and view the performance metrics of your TiDB cluster using the monitoring services.

```

File: tidb-cloud/third-party-integrations.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+## Required access
```
Comment (qiancai) [inline tidb-cloud/third-party-integrations.md:8]:
```suggestion

> **Note:**
>
> For [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier), third-party monitoring integrations are not supported.
 
## Required access
```

File: tidb-cloud/third-party-integrations.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+To edit third-party integration settings, you must have the `Organization Owner` access to your organization or `Project Member` access to the target project.
```
Comment (qiancai) [inline tidb-cloud/third-party-integrations.md:10]:
```suggestion
To edit third-party integration settings, you must have the `Owner` access to your organization or `Member` access to the target project.
```

File: tidb-cloud/third-party-integrations.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+## View or modify third-party integrations
+
+1. On the TiDB Cloud console, choose a target project that you want to view or modify, and then click the **Project Settings** tab.
+2. In the left pane, click **Integrations**. The available third-party integrations are displayed.
+
+## Available integrations
+
+### Datadog integration
+
+Configures TiDB Cloud to send metric data about your TiDB clusters to [Datadog](https://www.datadoghq.com/). You can view these metrics in your Datadog dashboards. To get a detailed list of all metrics that Datadog tracks, refer to [Datadog Integration](/tidb-cloud/monitor-datadog-integration.md).
```
Comment (qiancai) [inline tidb-cloud/third-party-integrations.md:21]:
```suggestion
With the Datadog integration, you can configure TiDB Cloud to send metric data about your TiDB clusters to [Datadog](https://www.datadoghq.com/) and view these metrics in your Datadog dashboards.

For the detailed integration steps and a list of metrics that Datadog tracks, refer to [Integrate TiDB Cloud with Datadog](/tidb-cloud/monitor-datadog-integration.md).
```

File: tidb-cloud/third-party-integrations.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+### Prometheus and Grafana integration
+
+Get a scrape_config file for Prometheus from TiDB Cloud and use the content from the file to configure Prometheus. You can view these metrics in your Grafana dashboards. To get a detailed list of all metrics that Prometheus tracks, refer to [Prometheus and Grafana Integration](/tidb-cloud/monitor-prometheus-and-grafana-integration.md).
```
Comment (qiancai) [inline tidb-cloud/third-party-integrations.md:25]:
```suggestion
With the Prometheus and Grafana integration, you can get a scrape_config file for Prometheus from TiDB Cloud and use the content from the file to configure Prometheus. You can view these metrics in your Grafana dashboards. 

For the detailed integration steps and a list of metrics that Prometheus tracks, see [Integrate TiDB Cloud with Prometheus and Grafana](/tidb-cloud/monitor-prometheus-and-grafana-integration.md).
```

File: tidb-cloud/third-party-integrations.md (hunk 3 above)
Comment (zhangyangyu) [reply tidb-cloud/third-party-integrations.md:2]: +1

File: TOC-tidb-cloud.md [hunk 11]
```diff
@@ -116,9 +116,7 @@
   - [Overview](/tidb-cloud/monitor-tidb-cluster.md)
   - [Built-in Monitoring](/tidb-cloud/built-in-monitoring.md)
   - [Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md)
-  - Third-Party Monitoring Integrations
-    - [Datadog Integration](/tidb-cloud/monitor-datadog-integration.md)
-    - [Prometheus and Grafana Integration](/tidb-cloud/monitor-prometheus-and-grafana-integration.md)
+  - [Third-Party Integrations](/tidb-cloud/third-party-integrations.md)
```
Comment (qiancai) [inline TOC-tidb-cloud.md:119]:
```suggestion
  - [Third-Party Monitoring Integrations](/tidb-cloud/third-party-monitoring-integrations.md)
```

File: tidb-cloud/third-party-integrations.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (qiancai) [inline tidb-cloud/third-party-integrations.md:1]: Would you please add "monitoring" to the filename? Thanks

Comment (hfxsd) [reply tidb-cloud/third-party-integrations.md]: Done

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**